can't build docs

Commit add9d6093bc48e4ae21b5378fe0b60b4d6715e2c broke `make html` on my mac. `make html` runs fine on the parent commit, whereas after the commit above I get:

```
whoburg$ make html
rm source/autodoc/*
sphinx-apidoc ../gpkit -o source/autodoc
Creating file source/autodoc/gpkit.rst.
Creating file source/autodoc/gpkit.constraints.rst.
Creating file source/autodoc/gpkit.interactive.rst.
Creating file source/autodoc/gpkit.nomials.rst.
Creating file source/autodoc/gpkit.tests.rst.
Creating file source/autodoc/gpkit.tools.rst.
Creating file source/autodoc/modules.rst.
sed -i '1i Glossary\n********\n\n*For an alphabetical listing of all commands, check out the* :ref:`genindex`\n' source/autodoc/gpkit.rst
sed: 1: "source/autodoc/gpkit.rst": bad flag in substitute command: 'c'
make: *** [apidoc] Error 1
```

Unfortunately I still get the sed warnings:

```
(gpkit)warrens-mbp-2:docs whoburg$ make html
./apidoc.sh
Creating file source/autodoc/gpkit.rst.
Creating file source/autodoc/gpkit.constraints.rst.
Creating file source/autodoc/gpkit.interactive.rst.
Creating file source/autodoc/gpkit.nomials.rst.
Creating file source/autodoc/gpkit.tests.rst.
Creating file source/autodoc/gpkit.tools.rst.
Creating file source/autodoc/modules.rst.
sed: 1: "source/autodoc/gpkit.rst": bad flag in substitute command: 'c'
sed: 1: "source/autodoc/gpkit.rst": bad flag in substitute command: 'c'
make: *** [apidoc] Error 1
```

--- a/docs/apidoc.sh
+++ b/docs/apidoc.sh
@@ -1,6 +1,9 @@
 rm source/autodoc/*
 sphinx-apidoc ../gpkit -o source/autodoc
 
-sed -i '1,3d' source/autodoc/gpkit.rst
+# Delete fist 3 lines
+sed -i'' -e '1,3d' source/autodoc/gpkit.rst
+
+# Add header
 header="Glossary\n********\n\n*For an alphabetical listing of all commands, check out the* :ref:\`genindex\`\n"
-sed -i "1i $header" source/autodoc/gpkit.rst
+echo $header | cat - source/autodoc/gpkit.rst > gpkit.rst && mv gpkit.rst source/autodoc/gpkit.rst

--- a/docs/apidoc.sh
+++ b/docs/apidoc.sh
@@ -1,3 +1,6 @@
 rm source/autodoc/*
 sphinx-apidoc ../gpkit -o source/autodoc
-sed -i '1i Glossary\n********\n\n*For an alphabetical listing of all commands, check out the* :ref:`genindex`\n' source/autodoc/gpkit.rst
+
+sed -i '1,3d' source/autodoc/gpkit.rst
+header="Glossary\n********\n\n*For an alphabetical listing of all commands, check out the* :ref:\`genindex\`\n"
+sed -i "1i $header" source/autodoc/gpkit.rst

--- a/docs/source/autodoc/gpkit.rst
+++ b/docs/source/autodoc/gpkit.rst
@@ -3,9 +3,6 @@ Glossary
 
 *For an alphabetical listing of all commands, check out the* :ref:`genindex`
 
-gpkit package
-=============
-
 Subpackages
 -----------
 

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -18,7 +18,7 @@ import os
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('../../../gpkit'))
 
 # -- General configuration ------------------------------------------------
 

